packages/perseus-core/src/data-schema.ts

```diff
@@ -1210,7 +1212,7 @@ export type PerseusNumericInputAnswer = {
     // NOTE: perseus_data.go says this is non-nullable even though we handle null values.
     maxError: number | null | undefined;
     // Unsimplified answers are Ungraded, Accepted, or Wrong. Options: "required", "correct", or "enforced"
-    simplify: string | null | undefined;
+    simplify: PerseusNumericInputSimplify;
```

Comment:
This could be a spicy move, but I think it's good to be more specific with our types here. While it resulted in some minor downstream updates, I think greater specificity will help us.

Reply:
Merging the `| null | undefined` into the type feels like it hides something important. Without referring to the type, this field looks required, but you find out that it's optional only when you inspect the new `PerseusNumericInputSimplify`. 

I'm not sure if we have a precedent, but I think I'd prefer we kept the new type as the union of string values and then add `| null | undefined` here so that the "optionality" stays in situ.